apps/meteor/app/cloud/server/functions/getWorkspaceAccessToken.ts

```diff
@@ -24,20 +24,18 @@ export async function getWorkspaceAccessToken(forceNew = false, scope = '', save
 		return '';
 	}
 
-	const workspaceCredentials = await WorkspaceCredentials.getCredentialByScope(scope);
-	if (!workspaceCredentials) {
-		throw new CloudWorkspaceAccessTokenError();
-	}
+	const scopes = scope === '' ? [] : [scope];
```

Comment:
Wouldn't this end up fetching token every time?
`WorkspaceCredentials.getCredentialByScopes([])` vs `WorkspaceCredentials.updateCredentialByScopes({scopes: <something else>, ...)`

Reply:
Oh yea, it should be the default: `workspaceScopes `